README.md

```diff
@@ -1,20 +1,236 @@
+
+## Destination Selection
+
+For every `source` there must also be a `destination`.  A `destination` is a JavaScript function that is located using selector syntax.  (Though it’s often less expressive since dot notation is usually enough to find the function.)  A simple example of a destination is `console.log`, which would print the output of a data layer to the console.
+
+Because `destination` is a function, this gives DLO a very flexible way to hand off data to a third party.  You can think of this hand off as DLO calling a specific JavaScript function with arguments.  The arguments are the data emitted from the data layer and anything added by `operators`.  In the simple `console.log` destination example, the object taken from the data layer would call `console.log(object)`, which would simply print the object on the console.
+
+A more practical example is calling FullStory’s [event](https://developer.fullstory.com/custom-events) function by setting the destination to `FS.event`.  There’s one catch though: the object emitted from the data layer must be the second argument for `FS.event`.  To add the first argument and move the object to the second, an operator will be used.  The next section talks about operators and the [Operator Tutorial](https://github.com/fullstorydev/fullstory-data-layer-observer/tree/main/docs/operator_tutorial.md) explains the process in more detail.
+
+## Data Layer Operators
+
+If a source emits data and a destination is the recipient, what makes the two compatible?  The answer is an `operator`.  An operator is designed to perform light transformation of an object emitted from the data layer so that it can be received by a destination.  Operators are chained together using the `operators` property in a rule.  Each operator performs a small transformation and provides the result to the next operator.  An operator can also choose to not pass along data (by returning `null`) at which point data handling stops and the destination is not called.
+
+While it may take a moment to familiarize yourself with operators, they are designed to do the work that would otherwise have been custom JavaScript.  Instead of writing code, you’ll configure operators to do the work.
+
+See the [Operator Tutorial](https://github.com/fullstorydev/fullstory-data-layer-observer/tree/main/docs/operator_tutorial.md) for an in-depth guide on using a few key operators.
+
+### Operator Reference
+
+| Name | Doc Link | Description |
+| ---- | -------- | ----------- |
+| convert | Doc | Formats a value to a bool, int, real, or string. |
+| flatten | Doc | Recursively flattens all properties into an object. |
+| function | Doc | Executes a function and returns the result. |
+| insert | Doc | Inserts a value into a list at a specified position. |
```

Comment:
How is the position specified?

Reply:
This is a property called `position`.  I still need to add the individual "doc" markdown files for each operator.  These will further explain operators with args and examples.

Reply:
I added a doc example for flatten.  Others should follow suit.

Reply:
All operator docs are now draft PRs.